aclk/agent_cloud_link.c

```diff
@@ -750,8 +752,8 @@ int aclk_execute_query(struct aclk_query *this_query)
         buffer_flush(local_buffer);
         local_buffer->contenttype = CT_APPLICATION_JSON;
 
-        aclk_create_header(local_buffer, "http", this_query->msg_id);
-
+        aclk_create_header(local_buffer, "http", this_query->msg_id, 0, 0);
+        buffer_strcat(local_buffer, ",\n\t\"payload\": ");
```

Comment:
This will create a duplicate payload as `aclk_create_header` already puts `"payload": ` in there

Reply:
Ignore this --  I saw you removed the payload from the `aclk_create_header`